external/fv3viz/examples/plot_cube_fortran_diagnostic.py

```diff
@@ -38,7 +38,7 @@ def get_web_dataset(url):
 
 fortran_diagnostic_ds = get_web_dataset(DATA_URL)
 
-gfdl_grid_metadata = GridMetadata("grid_xt", "grid_yt", "grid_x", "grid_y")
+gfdl_grid_metadata = GridMetadataFV3("grid_xt", "grid_yt", "grid_x", "grid_y")
```

Comment:
Great idea, I should definitely add one for scream. Per discussion in #2188, I'm going to try and see if it would be possible to keep native scream output format. Will circle back to this once we confirmed which format to use.

Reply:
Example added under `external/fv3viz/examples/plot_cube_scream_diagnostics.py`